components/dropbox/actions/download-file-to-tmp/download-file-to-tmp.mjs

```diff
@@ -0,0 +1,46 @@
+import dropbox from "../../dropbox.app.mjs";
+import common from "../common/common.mjs";
+import fs from "fs";
+import { file } from "tmp-promise";
+
+export default {
+  ...common,
+  name: "Download File To TMP",
```

Comment:
```suggestion
  name: "Download File to TMP",
```

Reply:
Changed capitalization to follow Google's Developer docs style guide

Reply:
Please address this issue